crux-http-server/src/crux/http_server.clj

```diff
@@ -380,14 +368,172 @@
       ((resolve 'crux.sparql.protocol/sparql-query) crux-node request)
       nil)))
 
+(defn link-all-entities
```

Comment:
Used an atom here to avoid having to pass around `links` through the recursive function - happy to change if it is a particular problem.

Reply:
as discussed, let's pass it down through the recursive calls